src/Sylius/Bundle/AdminBundle/Resources/views/Email/layout.html.twig

```diff
@@ -1,12 +1,12 @@
 {% block body %}
     {% autoescape false %}
-        {% set logo = channel.hostname is not null ? 'http://' ~ channel.hostname ~ asset('assets/admin/img/logo.png') : absolute_url(asset('assets/admin/img/logo.png')) %}
+        {% set logo = channel.hostname is not null ? 'https://' ~ channel.hostname ~ asset('assets/admin/img/logo.png') : absolute_url(asset('assets/admin/img/logo.png')) %}
```

Comment:
Only relevant changes are done in twig templates. What do you think about global twig config, which will toggle `http` and `https`? The default configuration should use https, but we will provide an easy way for backward compatibility.

Reply:
Not sure what you mean by global twig config. You mean adding a global twig variable containing this information?

Reply:
Adding a `scheme` attribute to the `Channel` entity could be also an idea. But it will require to add it to the admin form and default value should be `http` to ensure backward compatibility.

Reply:
Hoping that it's not too late...
Having it configurable per channel doesn't make much sense to me, because why would one have some channels on http and others on https? Besides that, it requires a lot more extra effort than adding a twig global variable.
I suggest adding an environment variable and use it's value for a Twig global variable.
Maybe also define an intermediate container parameter...